avocado/core/exceptions.py

```diff
@@ -85,6 +85,16 @@ class TestNotFoundError(TestBaseException):
     status = "NOT_FOUND"
 
 
+class NotATestError(TestBaseException):
```

Comment:
Now we have `NotATestError` and ` TestNotFoundError` both with same descriptions.